packages/vehicle-rental-overlay/package.json

```diff
@@ -18,9 +18,9 @@
     "url": "git+https://github.com/opentripplanner/otp-ui.git"
   },
   "dependencies": {
-    "@opentripplanner/core-utils": "^3.0.4",
     "@opentripplanner/from-to-location-picker": "^1.0.3",
     "@opentripplanner/zoom-based-markers": "^1.0.1",
```

Comment:
Should we bump zoom-based-markers here too?

Reply:
Yeah wasn't sure if we wanted to but have now bumped all deps in 11b29b2